src/lib.rs.in

```diff
@@ -674,7 +688,7 @@ class IronSdk {
     self_type IronOxide;
     private constructor = empty;
     static_method user_verify(_:&str) -> Result<Option<UserVerifyResult>, String>; alias userVerify;
-    static_method user_create(_:&str, _:&str) -> Result<UserCreateKeyPair, String>; alias userCreate;
+    static_method user_create(_:&str, _:&str, _:&UserCreateOpts) -> Result<UserCreateKeyPair, String>; alias userCreate;
```

Comment:
Are we making sure to name params now, or no? I thought the latest rust_swig used them in the generated code?

Reply:
Yes, I have now named the parameters and brought the documentation from ironoxide over